parsec/core/fs/workspacefs/workspacefs.py

```diff
@@ -630,7 +630,7 @@ async def _sync_by_id(
 
             # No new manifest to upload, the entry is synced!
             if new_remote_manifest is None:
-                return remote_manifest or (await self.local_storage.get_manifest(entry_id)).base
+                return remote_manifest or (await self.local_storage.get_manifest(entry_id)).base  # type: ignore[return-value]
```

Comment:
why the type ignore ?

Reply:
The ignore was not needed, It was a bad typing of the returned value